### PR TITLE
Marketing modal

### PR DESCRIPTION
**Pull request by mckayn10**

### CODE REVIEW

**Review by Tommydreamer57 — COMMENTED** [review]
It is much more readable to put each prop on one single line when you have more than one on a single element

**Review by Tommydreamer57 — COMMENTED** [review]
Also, is there a reason you added `-webkit-transition` and the other browser-specific properties to the `.modal-container` class (lines 30-33)? These are typically automatically added by webkit in create-react-app.

Also, you can use the `&` selector in scss to nest selectors that apply to the same element (lines 69-89) (yes this also works with pseudo-selectors)

File: src/components/MarketModal/MarketModal.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import './market-modal.scss';
+import $ from 'jquery';
```
Comment (huyqdang) [inline src/components/MarketModal/MarketModal.js]: it seems like Jquery got import but never used

**Comment by mckayn10** [comment]
I put each prop on it's own line

File: src/components/MarketTopNav/MarketTopNav.js [hunk 2]
```diff
@@ -1,16 +1,61 @@
...
+
+    handleChange = (key, value) => {
+        this.setState({
+            [key]: value
+        })
+    }
+
     render() {
+
+        console.log(this.state.username)
```
Comment (huyqdang) [inline src/components/MarketTopNav/MarketTopNav.js]: can you remove these console.logs

**Comment by mckayn10** [comment]
Also I had the webkit-transition because I copied the code from somewhere online. Fixed the scss file and used & for the selectors.

**Review by Tommydreamer57 — APPROVED** [review]
Looks good. You can also put the `:focus::placeholder` (line 79) inside the `:focus` block (line 82)

File: src/components/MarketTopNav/MarketTopNav.js [hunk 3]
```diff
@@ -37,8 +37,8 @@ class MarketTopNav extends Component {
 
     render() {
 
-        console.log(this.state.username)
-        console.log(this.state.password)
+        console.log(this.state.registerEmail)
```
Comment (huyqdang) [inline src/components/MarketTopNav/MarketTopNav.js:40]: these logs also need to be remove

**Pull request merged**